examples/chart/teleport/config/teleport.yaml

```diff
@@ -23,9 +26,10 @@ ssh_service:
 
 proxy_service:
   enabled: yes
-  public_addr: lens.gravitational.co:443
-  web_listen_addr: 0.0.0.0:8080
+#  public_addr: teleport.ngrok.donburi-apps.net:443
+  public_addr: teleport.example.com
```

Comment:
this should be a chart parameter I think

Reply:
Good point 👍  I was even considering to make the whole configuration provided as chart values, avoiding [the use of `{{ .Files.Get ... }}` in the template]({{ .Files.Get "config/teleport.yaml" | indent 4 }}). `Files.Get` works only for local charts. That's why you see [no `Files.Get`s for **user inputs** in the official Helm charts repository](https://github.com/kubernetes/charts/search?q=%22Files.Get%22&unscoped_q=%22Files.Get%22).

Reply:
I agree, let's move configuration straight to configmap then, so we can tweak it with values, or you think whole configfile can be a parameter?

Reply:
Addressed in the last commit